### PR TITLE
Framework support

### PR DESCRIPTION
**Pull request by robb**
This replaces the static library target with a dynamic Framework instead, CocoaPods remains unaffected but this makes binary distribution of a pre-built Nocilla.framework much easier.

### CODE REVIEW

**Comment by robb** [comment]
@jspahrsummers would you mind giving this a review?

File: Nocilla/Info.plist [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.alamofire.$(PRODUCT_NAME:rfc1034identifier)</string>
```
Comment (jspahrsummers) [inline Nocilla/Info.plist]: Is this the right org?

**Comment by jspahrsummers** [comment]
Looks pretty good, but the list of public headers doesn't seem to exactly align with the podspec. Is that a problem?

**Comment by ayanonagon** [comment]
Just tested this against Venmo’s integration using `0.35` of CocoaPods. (We haven’t started using `0.36` yet because bugs.) :octocat:

**Comment by robb** [comment]
:sparkling_heart:

**Pull request merged**